README.md

```diff
@@ -170,17 +178,23 @@ This is an application that connects to IoT Hub using the service SDK offering t
 
 ## Scenario Steps
 
-### 1. Review the existing samples
+### 1. Review the existing samples and docs
+
+If you need to get familiar with PnP concepts, take a time to familiarize yourself by reading the staging docs available in [review.docs.microsoft.com/azure/iot-pnp](https://review.docs.microsoft.com/azure/iot-pnp?branch=pr-en-us-114283)
+
+To learn how to create a PnP device we have tutorials for C, Node and Python:
 
-Follow the tutorial [Quickstart: Connect a sample IoT Plug and Play Preview device application running on Linux or Windows to IoT Hub (C)](https://review.docs.microsoft.com/azure/iot-pnp/quickstart-connect-device-c?branch=release-preview-refresh-iot-pnp) to build and run the existing [C device sample](https://github.com/Azure/azure-iot-sdk-c/tree/public-preview-pnp/digitaltwin_client/samples)
+Follow the tutorial [Quickstart: Connect a sample IoT Plug and Play Preview device application running on Linux or Windows to IoT Hub (C)](https://review.docs.microsoft.com/azure/iot-pnp/quickstart-connect-device-c?branch=pr-en-us-114283) to build and run the existing [C device sample](https://github.com/Azure/azure-iot-sdk-c/tree/public-preview-pnp/digitaltwin_client/samples)
```

Comment:
@ericmitt note I've updated the branch from the doc links

Reply:
great